pymchelper/readers/shieldhit.py

```diff
@@ -16,6 +16,153 @@ class SHBinaryReader:
     def __init__(self, filename):
         self.filename = filename
 
+    def test_version_0p6(self):
+        SH_BDO_MAGIC_NUMBER = b'xSH12A'
```

Comment:
better to use lowercase letters for constants.
In numpy to get pi number you type `np.pi`, not `np.PI`